src/MySqlConnector/Core/CachedProcedure.cs

```diff
@@ -23,13 +23,13 @@ internal sealed class CachedProcedure
 			{
 				try
 				{
-					using var cmd = connection.CreateCommand();
+					await using var cmd = connection.CreateCommand();
```

Comment:
This isn't supported before `netstandard2.1`, and isn't useful because `MySqlCommand` doesn't use asynchronous I/O when disposing.